### PR TITLE
Adds getPageVersion

### PR DESCRIPTION
**Pull request by cperryk**
Distantly related to [Trello](https://trello.com/c/Z0tqynOw/36-add-site-to-site-import-in-clay-cli)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
```
Comment (amycheng) [inline .gitignore:1]: lol

**Review by amycheng — CHANGES_REQUESTED** [review]
instead of "version", we should call it "state" (ie `getPageState`), since some time down the line, we might have versioning for pages.

Page states are: published, unpublished, and draft. 

Otherwise this looks good.

**Comment by cperryk** [comment]
@amycheng The name is consistent with `getComponentVersion`, which does the same thing for components. If we want to change them both to "state" instead of "version," that's fine, but that's a separate (and breaking) change.

**Comment by amycheng** [comment]
@cperryk blech. Okay, just add a comment somewhere that clarifies we're actually trying to get the published state so future developers trying to implement page versioning don't get confused.

**Review by amycheng — APPROVED** [review]

**Comment by jonwinton** [comment]
@amycheng there is no `unpublished` page state in the data, that's purely a concept of the page list.

The is only `editable`, `scheduled`, `published`

**Pull request merged**